src/lib.rs

```diff
@@ -1,12 +1,6 @@
 #![no_std]
 
-#[cfg(test)]
-#[macro_use]
-extern crate std;
-
-// #[cfg(test)]
-// use std::prelude::*;
```

Comment:
This was some trash that I accidentally committed after trying to integrate a mocking library that required `std`.